### DISCUSSION TITLE
【求助】难度70打不过了！我的暑假作业副本！

### DISCUSSION THREAD

**Discussion by DrAshewell** [share121/inter-knot#2477] [其他]
有没有人接“代写暑期作业”委托，报酬十万丁尼，球球了！
![d3598e250a143e37](https://github.com/user-attachments/assets/7e3577a9-b8cb-4577-9662-b22aa6b2ed16)

**Comment by ZHOUZHOU567** [comment]
这个委托我不接。

**Comment by xjf0326** [comment]
**_### 5000菲林我接_**

**Comment by cza2019** [comment]
我出4999菲林

**Comment by 00zzxx** [comment]
有空在绳网上水不如赶快能写一点是一点，还有一种可能，说不定老师不收呢<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by hua-yuanshuai** [comment]
这委托我....我不接<img src="https://img-static.mihoyo.com/communityweb/upload/1a25d24ab81aaa88543ffad7cbf3f944.png" alt="绝区零_暗中观察" width="75" >

**Comment by CloudSwordSage** [comment]
学聪明点，我赌他不检查<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by yuxigongzi** [comment]
暑假作业就两种，一种是老师看，这种暑假已经让学生通过钉钉什么的上传批改了，另一种就是不看，最后收一下也不批改也不讲评，所以建议随便混混得了

**Comment by Aha-SWYH** [comment]
这委托……谁爱接谁接
